src/base-apis.js

```diff
@@ -26,6 +29,17 @@ limitations under the License.
 const httpApi = require("./http-api");
 const utils = require("./utils");
 
+function termsUrlForService(serviceType, baseUrl) {
+    switch (serviceType) {
+        case SERVICETYPES.IS:
+            return baseUrl + httpApi.PREFIX_IDENTITY_V2;
+        case SERVICETYPES.IM:
+            return baseUrl + '/terms/';
```

Comment:
A strict reading of the MSC would seem to say `GET $prefix/terms` without a trailing slash after `terms`. (Unsure how much this matters in practice for our APIs.)